Application/EdFi.Ods.Api.IntegrationTestHarness/Program.cs

```diff
@@ -26,33 +23,48 @@ public static async Task Main(string[] args)
             var _logger = LogManager.GetLogger(typeof(Program));
             _logger.Debug("Loading configuration files");
 
-            var executableAbsoluteDirectory = Path.GetDirectoryName(typeof(Program).Assembly.Location);
-
-            ConfigureLogging(executableAbsoluteDirectory);
+            ConfigureLogging();
 
             var host = Host.CreateDefaultBuilder(args)
                 .ConfigureAppConfiguration(
-                    (hostBuilderContext, configBuilder) =>
+                    (hostingContext, config) =>
                     {
-                        string appSettingsPath = Path.Combine(executableAbsoluteDirectory,"appsettings.json");
+                        var env = hostingContext.HostingEnvironment;
+
+                        config
+                            .SetBasePath(Directory.GetParent(AppContext.BaseDirectory).FullName)
+                            .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                            .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true, reloadOnChange: true)
+                            .AddJsonFile("appsettings.user.json", optional: true, reloadOnChange: true);
+
```

Comment:
remove extra line

Reply:
should we abstract this into an extension method into EdFi.Ods.Api?

Reply:
I'm going to add this to my list of tickets to create.